relay-filter/src/browser_extensions.rs

```diff
@@ -43,7 +43,9 @@ static EXTENSION_EXC_VALUES: Lazy<Regex> = Lazy::new(|| {
         webkit-masked-url:|
         # Firefox message when an extension tries to modify a no-longer-existing DOM node
         # See https://blog.mozilla.org/addons/2012/09/12/what-does-cant-access-dead-object-mean/
-        can't\saccess\sdead\sobject
+        can't\saccess\sdead\sobject|
+        # Crypocurrency related extension errors
+        Cannot\sredefine\sproperty:\s(solana|ethereum)
```

Comment:
Should we include `*wallet*` matches too? Not as many matches with `"%Cannot redefine property:%wallet%"`, but it's another improvement.

Reply:
Let me follow up with the team, i think i'd like to continue matching very specific words like okxwallet